.github/workflows/docker_all-modules.yml

```diff
@@ -0,0 +1,38 @@
+# This is a workflow to build and optionally push all module Docker images
+#
+# This workflow will run only on manual triggers
+
+name: Build all Docker images
+on:
+  workflow_dispatch:
+    inputs:
+      push-ecr:
+        description: "Push to AWS ECR"
+        type: boolean
+        required: false
+
+jobs:
+  build-docker-modules:
+    name: Build Docker Images
+    strategy:
+      fail-fast: false
+      matrix:
+        # add all modules with active docker images here
+        module:
+          - hello-python
+          - simulate-sce
```

Comment:
Here's where we'd keep adding modules, correct? (This was kind of answered by your reorg that wiped out a comment 😄 )

Reply:
Yes, I added a comment here too because I knew it would be unclear. I'd add another comment at the top to tell people where to go.